.github/workflows/master_build_merge.yml

```diff
@@ -241,13 +301,139 @@ jobs:
             docker push ${DOCKERHUB_ORG}/$IMAGE:${LATEST_TAG}
             set +e
           }
+          build() {
+            type="$1"
```

Comment:
Same here.  Since we are building generic framework. All functions must validate its input parameters for what their values are expected to be.